### ISSUE TITLE
Point of Sales : Discount Didn't reset

### ISSUE THREAD

**Issue by thida-bcn** [frappe/erpnext#33276]
### Information about bug

1) go to point of sales
create the first voucher with a discount % of 10

![image](https://user-images.githubusercontent.com/62122640/206598805-41caa085-b25c-4624-91ff-abc52bc46ccf.png)

complete order
![image](https://user-images.githubusercontent.com/62122640/206598842-b3fb49a6-c3e6-41ca-a75c-04eda8bc49a0.png)

2) new order
The discount % applied at the previous invoice didn't reset on the screen
it should reset to 0
![image](https://user-images.githubusercontent.com/62122640/206598880-7967c702-bbc1-4944-9a3d-31c74f4adf9c.png)
![image](https://user-images.githubusercontent.com/62122640/206598923-50a4bb32-88d5-408e-8850-3270ae0c6c46.png)
![image](https://user-images.githubusercontent.com/62122640/206598961-13523247-df42-4531-9f57-d815aaec31c1.png)

### Module

selling

### Version

erpnext 13.41.1
frappe 13.43.2

### Installation method

manual install

**Comment by dj12djdjs** [comment]
Makes sense, the voucher for the previous sale should not apply for the current unless operator re-applies the voucher.
